cli/cmd/transformCmd.go

```diff
@@ -39,6 +40,8 @@ func init() {
 
 	transformCmd.Flags().StringVarP(
 		&input, "input", "i", "", "input file (optional; if not specified, stdin/pipe is used)")
+	transformCmd.Flags().BoolVarP(
+		&ndjson, "stream", "", false, "change the output format to ndjson")
```

Comment:
change the argument help display text to `"if specified, each record will be a standalone/full JSON blob and printed out immediately once transform is done"`

Reply:
again, `ndjson` -> `stream`